modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -65,7 +65,7 @@ Specify `yes` to compress the message or `no` to not compress. The default is `y
 include a unit: "year", "month", "week", "day", "h" or "m". Enter `0` to disable. The default is `1month`.
 <5> Configure rate limiting. If more logs are received than what is specified in `RateLimitBurst` during the time interval defined by `RateLimitIntervalSec`, all further messages within the interval are dropped until the interval is over. It is recommended to set `RateLimitIntervalSec=30s` and `RateLimitBurst=10000`, which are the defaults.
 <6> Specify how logs are stored. The default is `persistent`:
-* `volatile` to store logs in memory in `/var/log/journal/`.
+* `volatile` to store logs in memory in `/run/log/journal/`.
```

Comment:
```suggestion
* `volatile` to store logs in memory in `/run/log/journal/`. These logs are lost after rebooting.
```
Do you mean something like this @kabirbhartiRH ?

Reply:
@abrennan89 Yes :)